[2019-10-10 00:43:01,500] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:31: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

[2019-10-10 00:43:01,519] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:39: The name tf.train.exponential_decay is deprecated. Please use tf.compat.v1.train.exponential_decay instead.

[2019-10-10 00:43:01,526] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/venv/lib/python3.7/site-packages/tensorflow/python/ops/init_ops.py:1251: calling VarianceScaling.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
[2019-10-10 00:43:01,526] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:295: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dense instead.
[2019-10-10 00:43:01,689] WARNING: Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x13ae1fb90>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x13ae1fb90>>: AssertionError: Bad argument number for Name: 3, expecting 4
[2019-10-10 00:43:01,694] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/venv/lib/python3.7/site-packages/tensorflow/python/ops/nn_impl.py:180: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
[2019-10-10 00:43:01,708] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:119: The name tf.train.AdamOptimizer is deprecated. Please use tf.compat.v1.train.AdamOptimizer instead.

[2019-10-10 00:43:01,709] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:122: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

[2019-10-10 00:43:01,709] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:122: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

[2019-10-10 00:43:01,808] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:130: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

[2019-10-10 00:43:01,808] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:134: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

[2019-10-10 00:43:01,871] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:137: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

[2019-10-10 00:43:01,888] INFO: fit on 9630 sample
[2019-10-10 00:43:04,602] INFO: [epoch-0, batch-0] -- Train Loss: 0.652727 NDCG: 0.327222 (0.407905) ERR: 0.211234  -- Valid Loss: 0.658822 NDCG: 0.377251 (0.443870) ERR: 0.213639 -- 2.713958 s
[2019-10-10 00:43:04,602] INFO: epoch: 1
[2019-10-10 00:43:04,728] INFO: epoch: 2
[2019-10-10 00:43:07,600] INFO: [epoch-2, batch-100] -- Train Loss: 0.627223 NDCG: 0.292490 (0.379591) ERR: 0.211234  -- Valid Loss: 0.642901 NDCG: 0.333902 (0.406719) ERR: 0.213639 -- 5.711331 s
[2019-10-10 00:43:07,644] INFO: epoch: 3
[2019-10-10 00:43:10,602] INFO: [epoch-3, batch-200] -- Train Loss: 0.601836 NDCG: 0.350812 (0.422713) ERR: 0.211234  -- Valid Loss: 0.615927 NDCG: 0.395027 (0.455326) ERR: 0.213639 -- 8.713418 s
[2019-10-10 00:43:10,626] INFO: epoch: 4
[2019-10-10 00:43:13,803] INFO: [epoch-4, batch-300] -- Train Loss: 0.588401 NDCG: 0.383230 (0.448310) ERR: 0.211234  -- Valid Loss: 0.602461 NDCG: 0.423085 (0.478453) ERR: 0.213639 -- 11.915222 s
[2019-10-10 00:43:13,807] INFO: epoch: 5
[2019-10-10 00:43:13,872] WARNING: From /Users/neo1/PycharmProjects/vec4irltr/ltr/model.py:143: The name tf.train.write_graph is deprecated. Please use tf.io.write_graph instead.

[2019-10-10 00:43:13,943] INFO: /Users/neo1/PycharmProjects/vec4irltr/lr/weights/lr/model.checkpoint is not in all_model_checkpoint_paths. Manually adding it.
[2019-10-10 00:43:14,125] WARNING: Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x13b8fb2d0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x13b8fb2d0>>: AssertionError: Bad argument number for Name: 3, expecting 4
[2019-10-10 00:43:14,306] INFO: fit on 9630 sample
[2019-10-10 00:43:17,164] INFO: [epoch-0, batch-0] -- Train Loss: 0.652727 NDCG: 0.327222 (0.407905) ERR: 0.211234  -- Valid Loss: 0.658822 NDCG: 0.377251 (0.443870) ERR: 0.213639 -- 2.858028 s
[2019-10-10 00:43:17,164] INFO: epoch: 1
[2019-10-10 00:43:17,280] INFO: epoch: 2
